runners/zachjs-sv2v

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+zachjs-sv2v $1
```

Comment:
Will this not be problematic when we do #54 ?

I understand the need to include "zachjs" in the runner name, but I am not sure about the binary name.